test/SignIn.test.js

```diff
@@ -0,0 +1,115 @@
+const { cleanDB } = require('../app/services/users');
+const jwt = require('jsonwebtoken');
+
+describe('Sign In endpoint test', () => {
+  afterEach(() => cleanDB());
+
+  it('should return no input email', async () => {
+    const userCredentials = {
+      email: undefined,
+      password: undefined
+    };
+
+    const response = await request(app)
+      .post('/users/sessions')
+      .send(userCredentials);
+    expect(response.statusCode).toBe(400);
+    expect(response.text).toBe('No input email!');
+  });
+
+  it('should return no input password', async () => {
+    const userCredentials = {
+      email: 'rodrigo.videla@woflox.com.ar',
+      password: undefined
+    };
+
+    const response = await request(app)
+      .post('/users/sessions')
+      .send(userCredentials);
+    expect(response.statusCode).toBe(400);
+    expect(response.text).toBe('No input password!');
+  });
+
+  it('should return not a valid WOLOX email', async () => {
+    const userCredentials = {
+      email: 'rodrigo.videla@woflox.com.ar',
+      password: 'password99'
+    };
+
+    const response = await request(app)
+      .post('/users/sessions')
+      .send(userCredentials);
+    expect(response.statusCode).toBe(200);
+    expect(response.text).toBe(`The email: ${userCredentials.email} is not a valid WOLOX email.`);
+  });
+
+  it('should return email not registered', async () => {
+    const userCredentials = {
+      email: 'rodrigo.videla@wolox.com.ar',
+      password: 'password99'
+    };
+
+    const response = await request(app)
+      .post('/users/sessions')
+      .send(userCredentials);
+    expect(response.statusCode).toBe(500);
+    expect(response.text).toBe(`The email: ${userCredentials.email} is not registered.`);
+  });
+
+  it('should return the password was wrong', async () => {
+    const user = {
+      firstName: 'rodrigo',
+      lastName: 'videla',
+      email: 'rodrigo.videla@wolox.com.ar',
+      password: 'password99'
+    };
+
+    const userCredentials = {
+      email: 'rodrigo.videla@wolox.com.ar',
+      password: 'password991'
+    };
+
+    const response = await request(app)
+      .post('/users')
+      .send(user);
+    expect(response.statusCode).toBe(200);
+    expect(response.text).toBe(`The user ${user.firstName} ${user.lastName} was successfully created.`);
+
+    const response2 = await request(app)
+      .post('/users/sessions')
+      .send(userCredentials);
+    expect(response2.statusCode).toBe(200);
+    expect(response2.text).toBe(
+      `The password for the user with the email: ${userCredentials.email} was wrong.`
+    );
+  });
+
+  it('should return the token', async () => {
+    const user = {
+      firstName: 'rodrigo',
+      lastName: 'videla',
+      email: 'rodrigo.videla@wolox.com.ar',
+      password: 'password99'
+    };
+
+    const userCredentials = {
+      email: 'rodrigo.videla@wolox.com.ar',
+      password: 'password99'
+    };
+
+    const response = await request(app)
```

Comment:
Same here. Delete this and insert the user with the factory.

Reply:
Also keep in mind is not ok to make assertions about two different features or functions in the same test. You either test for the user creation or for the toke signIn request, not both.

If you need to make test for the user creation endpoint then test it separately